### PR TITLE
Feature/lock whitelisted tokens

### PR DESCRIPTION
**Pull request by anxolin**
Dima, I'm working on this branch for building a price feed. It's not ready yet, but what I have now is mergeable and I also fixed the docker problem you are experiencing.

If you approve, we can merge it (but not delete the branch), so I can continue working in this branch.

What do you think?

### CODE REVIEW

File: contracts/FixedPriceOracle.sol [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    }
+    
+    function setPrices(
+        address[] tokens,
+        uint[] numerators,
+        uint[] denominators
+    ) public onlyOwner notFrozen {
+        for (uint i = 1 ; i < tokens.length ; i++) {
+            address token = tokens[i];
+            uint numerator = numerators[i];
```
Comment (Velenir) [inline contracts/FixedPriceOracle.sol]: Any reason you don't do `setPrice(tokens[i], numerators[i], denominators[i]);` here? Is it more gas-efficient to inline?

File: migrations/11_SCHEME-REP-Lock_Whitelisted_tokens.js [hunk 2]
```diff
@@ -23,8 +25,12 @@ module.exports = async function (deployer) {
   const dxController = await DxController.deployed()
   const gnoToken = await GnoToken.deployed()
 
-  console.log('Deploy DxLockGno4Rep that inherits from LockingToken4Reputation')
-  const dxLockGno4Rep = await deployer.deploy(DxLockGno4Rep)
+
+  console.log('Deploy FixedPriceOracle for setting the prices for the tokens')
+  const fixedPriceOracle = await deployer.deploy(FixedPriceOracle)
```
Comment (Velenir) [inline migrations/11_SCHEME-REP-Lock_Whitelisted_tokens.js]: Why are you deploying `FixedPriceOracle` here and not using it anywhere? Will it be passed into `DxLockWhitelisted4Rep` when `LockWhitelisted4Rep` is ready?

File: truffle.js [hunk 3]
```diff
@@ -2,8 +2,12 @@ const assert = require('assert')
 const HDWalletProvider = require('truffle-hdwallet-provider')
 const DEFAULT_GAS_PRICE_GWEI = 5
 const GAS_LIMIT = 6.5e6
+
 const DEFAULT_MNEMONIC = 'candy maple cake sugar pudding cream honey rich smooth crumble sweet treat'
 
+// Load env vars
+require('dotenv').load()
```
Comment (Velenir) [inline truffle.js]: Don't use `.load` use `.config`. Load is deprecated and still there for backwards compatibility.

File: contracts/FixedPriceOracle.sol (hunk 1 above)
Comment (anxolin) [reply contracts/FixedPriceOracle.sol]: Yes, so I don't have to access the arrays 2 times. Notice the emit of the event after that.

**Pull request merged**